apps/judicial-system/backend/src/app/messages/notifications.ts

```diff
@@ -336,7 +336,7 @@ export const notifications = {
     },
     bodyV3: {
       id:
-        'judicial.system.backend:notifications.prison_court_date_email.body_v3',
+        'judicial.system.backend:notifications.prison_court_date_email.body_v4',
```

Comment:
Any reason to change this key?

Reply:
Yes, this string uses `judicial.system.backend:notifications.prison_court_date_email.request_textv2` and this string did not update until I changed this id.